client/extensions/woocommerce/state/sites/promotions/helpers.js

```diff
@@ -15,6 +15,22 @@ export function createPromotionFromProduct( product ) {
 	};
 }
 
+export function createProductUpdateFromPromotion( promotion ) {
+	const { productIds } = promotion.appliesTo;
```

Comment:
Should we check for `promotion` here or maybe default the argument to `{}`?

Reply:
This is in the category of "should never happen", but sure, I'll add it.

Reply:
Ha, kind of figured as much - feel free to disregard then.